jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc

```diff
@@ -8,13 +8,15 @@
 *******************************************************************
 ////
 
-[[expert_group]]
-=== Expert Group Members
+[[project_team]]
+=== Project Team Members
 
-This specification is being developed as part of JSR 370 under the Java
-Community Process. It is the result of the collaborative work of the
-members of the JSR 370 Expert Group. The following are the present
-expert group members:
+This specification is being developed as part of Jakarta EE under the
+Jakarta EE Specification Process. It is the result of the collaborative work
```

Comment:
```suggestion
This specification is being developed as part of Jakarta RESTful Web Services project under the
Jakarta EE Specification Process. It is the result of the collaborative work
```

I think it is fine as it is written, but thought it might be good to state the project name here since I think it maps more directly to the JSR, and it might sound better than having "Jakarta EE" twice in the same sentence.

Reply:
Good idea! I just applied the suggested change!